shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -198,6 +206,24 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
         return super.onKeyDown(keyCode, event);
     }
 
+    @Override
+    protected void onDraw(Canvas canvas) {
+        //super.onDraw(canvas);
+        if (bmap != null) {
+            canvas.drawBitmap(bmap, new Matrix(), new Paint());
+        }
+    }
+
+    public void updateSoftwareBuffer(ByteBuffer buffer, int width, int height) {
+
+        bmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
+        bmap.copyPixelsFromBuffer(buffer);
+        postInvalidate();
+        if (!isSoftware) {
```

Comment:
If we are not using software rendering, we should have never called this method. We can get rid of this setter.